interfaces/system_key_test.go

```diff
@@ -226,3 +238,23 @@ func (s *systemKeySuite) TestInterfaceSystemKeyMismatchVersions(c *C) {
 	_, err = interfaces.SystemKeyMismatch()
 	c.Assert(err, Equals, interfaces.ErrSystemKeyVersion)
 }
+
+func (s *systemKeySuite) TestStaticVersion(c *C) {
+	// this is a static check to ensure we remember to bump the
+	// version when we add fields
+	//
+	// *** IF THIS FAILS, YOU NEED TO BUMP THE VERSION BEFORE "FIXING" THIS ***
+	var sk interfaces.SystemKey
+	c.Check(reflect.ValueOf(sk).NumField(), Equals, interfaces.SystemKeyVersion)
```

Comment:
Wouldn't this fail when we remove a field? Also, when we drop a field and then add one, version should be bumped twice IMO.

Reply:
yes, it would fail when we remove a field, and when we change a field. We'll have to fix the test for that.
And yes, about bumping it twice. This check is intended to remind us to bump things, not necessarily to catch every instance of needing to bump things --- that's why the next check is also part of the test.

Reply:
I think we should add comments to this effect.
spread's already underway. If it succeeds, and nothing needs actually fixing, can you add them in your followup?

Reply:
Yeah, will do.